lib/clearance/configuration.rb

```diff
@@ -124,9 +132,18 @@ def initialize
       @rotate_csrf_on_sign_in = true
       @routes = true
       @secure_cookie = false
+      @signed_cookie = false
       @sign_in_guards = []
     end
 
+    def signed_cookie=(value)
+      if [true, false, :migrate].include? value
+        @signed_cookie = value
+      else
+        raise "unexpected signed_cookie; allowed: true/false/:migrate"
```

Comment:
Is there a way we could make this exception clearer? I'm concerned that if this were in a stack trace somewhere it wouldn't be clear to a user where it was coming from or how to fix it.

Would something like

"Clearance's `signed_cookie` configuration value is invalid. Valid values are `true`, `false`, or `:migrate`. Set this option via Clearance.configure in an initializer"

Reply:
Sounds good to me.